code/game/objects/items/storage/marine_boxes.dm

```diff
@@ -17,6 +17,7 @@
 	new /obj/item/weapon/gun/rifle/standard_smartmachinegun(src)
 	new /obj/item/ammo_magazine/standard_smartmachinegun(src)
 	new /obj/item/ammo_magazine/standard_smartmachinegun(src)
+	new /obj/item/ammo_magazine/standard_smartmachinegun(src)
```

Comment:
This wasn't mentioned anywhere.

Reply:
Because it's moved from points to essential kit?
So not a change at all